test/core/security/grpc_audit_logging_test.cc

```diff
@@ -81,26 +82,82 @@ class AuditLoggingTest : public ::testing::Test {
 
 }  // namespace
 
-TEST_F(AuditLoggingTest, SuccessfulLoggerCreation) {
+//
+// AuditLoggerRegistryTest
+//
+
+TEST_F(AuditLoggerRegistryTest, SuccessfulLoggerCreation) {
   auto result = AuditLoggerRegistry::ParseConfig(kName, Json());
   ASSERT_TRUE(result.ok());
   ASSERT_NE(AuditLoggerRegistry::CreateAuditLogger(std::move(result.value())),
             nullptr);
```

Comment:
I know this test wasn't added in this PR, but I noticed that this functionally tests
1. `ParseConfig` returns something with `result.ok()`
2. `CreateAuditLogger` returns something that is not `nullptr`

Are there additional checks that would be worthwhile here? I know other tests cover more detail, but if they do is this really even worth having?

Reply:
This test is to check third-party logger factory registration and logger creation. It's somewhat different from stdout logger that we register in the ctor of the registry. So I think it's still needed. I also don't think there is anything else needed since we don't test the `TestAuditLoggerFactory` itself.